src/content/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest.mdx

```diff
@@ -1,114 +1,257 @@
+        "plugin": {
+        "source": "kubernetes",
+        "type": "fluent-bit",
+        "version": "1.8.1"
+        },
+        "stream": "stderr",
+        "time": "2021-09-14T12:30:49.138824971Z",
+        "timestamp": 1631622649138
+    }
+    ]
+    ```
+
+    This is how the previous log record would look like after enabling `lowDataMode`:
+
+    ```json
+    [
+    {
+        "cluster_name": "api-test",
+        "container_name": "newrelic-logging",
+        "namespace_name": "nrlogs",
+        "pod_name": "nri-bundle-newrelic-logging-jxnbj",
+        "message": "[2021/09/14 12:30:49] [ info] [engine] started (pid=1)\n",
+        "stream": "stderr",
+        "timestamp": 1631622649138
+    }
+    ]
+    ```
+    </Collapser>
+    <Collapser
+        id="nr-pixie-integration"
+        title="New Relic Pixie integration"
+    >
+
+    If you enable `lowDataMode`, the `newrelic-pixie` integration performs heavier sampling on Pixie spans and collect data less often, from 10 seconds to 15 seconds. These are the `lowDataMode` settings:
+
+    ```
+    HTTP_SPAN_LIMIT: 750
+    DB_SPAN_LIMIT: 250
+    COLLECT_INTERVAL_SEC: 15
+    ```
+
+    You can find the default settings for these parameters and others in the [newrelic-pixie-integration](https://github.com/newrelic/newrelic-pixie-integration) Github repo.
+
+    </Collapser>
+</CollapserGroup>
+
+
+## Reduce data ingested by changing the scrape interval [#scrape-interval]
+
+The New Relic Kubernetes integration allows changing the interval at which metrics are gathered from the cluster. This lets you choose the right balance between data resolution and usage. We suggest choosing an interval between 15 and 30 seconds for the best experience.
+
+The modification of this interval will depend on the method used to install Kubernetes:
+
+    * **Guided install** (New Relic CLI, Helm, Manifest). You can't modify this value following our guided install flow.
+
+    * **Helm**. Set `interval` in the [`nri-bundle` chart](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) the value you want.
+
+    * **Manifest**. Set `interval` in the Manifest configuration yaml file to the value you want. If you followed our [installation instructions](/install/kubernetes/?dropdown1=manifest), the file was called `newrelic-manifest.yaml`.
+
+If you want to change the scrape interval, just add the interval value under the `newrelic-infrastructure` section. Take a look at this example of the `values-newrelic.yaml` file to see how it looks:
+
+```yaml
+global:
+  licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+  cluster: _K8S_CLUSTER_NAME_
+
+# ... Other settings
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings
+  common:
+    config:
+      interval: 25s
 ```
-- kube_
-- container_
-- machine_
-- cadvisor_
+
+<Callout variant="important">
+    Setting `interval` to values larger than `40s` isn't allowed.
```

Comment:
Modified the note:
Note that if you're using the [cluster explorer](/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#cluster-explorer-use) instead of the [Kubernetes navigator](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer/#navigator-preview), you're not allowed to set `interval` to values greater than `40s`.